tools/fastlane-plugin/lib/fastlane/plugin/bugsnag/actions/upload_symbols_to_bugsnag.rb

```diff
@@ -126,13 +127,22 @@ def self.available_options
       private
 
       def self.default_info_plist_path
```

Comment:
I would really like for this plist/config file logic to be in a shared class. This level of copy pasta is not ideal. I would recommend a separate PR into this one to make it easy to review if you feel like taking that on @xander-jones

Reply:
New ticket created to look into this (PLAT-4743)